packages/template/src/App.vue

```diff
@@ -6,10 +6,9 @@
         :active-route="router.currentRoute.value.path"
       >
         <template #externals>
-          <div class="dropdown-option mb-3">
-            <span> Support </span>
-            <SpExternalArrowIcon />
-          </div>
+          <SpAccDropdownExternalLink title='Support' link=''/>
```

Comment:
Let's make links point to Starport properties by default, so https://discord.gg/ignt, https://twitter.com/starporthq, https://starport.com

Reply:
Do you mean like this:
```
<SpAccDropdownExternalLink title='Discord' link='https://discord.gg/ignt'/>
<SpAccDropdownExternalLink title='Twitter' link='https://twitter.com/starporthq'/>
<SpAccDropdownExternalLink title='Starport' link='https://starport.com/'/>
```